en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/P/Prismatic_Wall.md

```diff
@@ -21,7 +40,7 @@ Once a layer is destroyed, it is gone for the duration of the spell.
 To pass or reach through the barrier a creature does so one layer at a time and must make a Dexterity saving throw for each layer or be subjected to that layer’s effects.
 On a successful save, any damage taken from a layer is reduced by half.
 
-A Rod of Cancellation can destroy a Prismatic Wall, but an [Antimagic Field](#Antimagic_Field_antimagic_field) has no effect.
+An [Antimagic Field](#Antimagic_Field_antimagic_field) has no effect on the wall.
```

Comment:
The _Rod of Cancellation_ is not a 5e item and it was removed from this spell in [an errata](https://media.wizards.com/2016/downloads/DND/PH-Errata.pdf).